src/jdk.compiler/share/classes/com/sun/tools/javac/resources/ct_ja.properties

```diff
@@ -0,0 +1,770 @@
+#
```

Comment:
The change to this file (moving ct.properties to ct_ja.properties) does look suspicious. Looks like this is not a translation file, but some kind of a configuration for `javac`. If so, it should not be translated (and translation configuration in the closed repository should exclude this file)